Comparison Engine/Comparison Engine/Child Forms/DrinkForm.cs

```diff
@@ -11,41 +12,67 @@ namespace Comparison_Engine.Forms
 {
     public partial class DrinkForm : Form
     {
-        public DrinkForm(Drink drink)
+        public Drink drink = null;
+        public BarManager barManager = null;
+        public MapForm mapForm = null;
+        public MapController mapController = null;
+        public DrinkForm(Drink drnk, BarManager barMan, MapForm mainMapForm, MapController mainMapController)
```

Comment:
Misspelled "drink"?

Reply:
I wrote it down as "drnk" so it wouldn't conflict with the already present variable "drink"

Reply:
Changed "drnk" to "tempDrink" for clarity